sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,9 @@
+select "genres"."name" as "genre",
+  count ("castMembers".*) as "filmCount"
+  from "castMembers"
+    join "actors" using ("actorId")
+    join "films" using ("filmId")
+    join "filmGenre" using ("filmId")
+    join "genres" using ("genreId")
+    where "castMembers"."actorId" = 178
```

Comment:
Please filter by the criteria in the spec, there's no guarantee the actor you're looking for will have that id.

---

![image](https://user-images.githubusercontent.com/16143771/180361961-ce9f8ca8-dcc9-4e47-9653-d6eff08dc0ce.png)

Reply:
![image](https://user-images.githubusercontent.com/97073517/180362847-495640af-476a-4de2-a9fe-29cfb9062aea.png)

![image](https://user-images.githubusercontent.com/97073517/180362866-49ac31b9-6ab9-40a1-bebf-fe51822b7a80.png)